simple_parsing/wrappers/field_wrapper.py

```diff
@@ -687,10 +684,15 @@ def required(self, value: bool):
     @property
     def type(self) -> Type[Any]:
         """Returns the wrapped field's type annotation."""
-        if self._type is not None:
-            return self._type
-        else:
+        if self._type is None:
             self._type = self.field.type
+            
+            if self.is_choice and self.choice_dict:
+                if utils.is_list(self.field.type):
+                    self._type = List[str]
```

Comment:
Interesting, I didn't think of doing it that way.
I think it would ideally be better not to do it through changing `self._type`, because this indirectly depends on the other part of the code that constructs the `arg_options` dict that gets passed to `add_argument`.

For now you can leave this as-is, and I'll think about it some more.